src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,30 @@
 import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import { PreparedPost } from '../../types/PreparedPost';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface Props {
+  post: PreparedPost
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
```

Comment:
Consider removing the extra space before 'Posted by' to keep the text consistent with the rest of the code. 🧹